sympy/polys/domains/finitefield.py

```diff
@@ -232,6 +232,14 @@ def __init__(self, mod, symmetric=True):
     def tp(self):
         return self._tp
 
+    @property
+    def is_Field(self):
```

Comment:
I think that it's first time we are starting to overriding `is_Field` even though this still subclasses from `Field` though.

Reply:
Yes, but `isinstance(K, Field)` isn't used anywhere. The proper solution is that this class should be two classes.